src/commands/fng.py

```diff
@@ -0,0 +1,49 @@
+from discord import app_commands, Interaction
+
+
+class FNGGroup(app_commands.Group):
+    def __init__(self):
+        super().__init__(name="fng", description="FNG Group")
```

Comment:
Hi Ray,
Could the description here be the same as the description for the command ("Outputs the date of the next Friday Night Games with food")? I'm thinking this because of how this group will only have one command.

Reply:
If we add more commands related to FNG then the description would have to be changed and it's also a duplicate of the command that we have. I think instead, it should just describe the overall group, which is providing information about Friday Night Games. Something like, "Provides information about Friday Night Games (FNG)" would be good.

I think we should also have a command to explain what Friday Night Games is about, although that could be added to the `FAQ` command group (@phoenixpereira what are your thoughts on this?)

Reply:
Yeah for now it would be good if you can add a seperate command to the `FAQ` command group to explain what Friday Night Games is. Implementing a FAQ system as described in #2 would take some time.

Reply:
@ChristianDianos can you implement this as well?